ntbs-service/Services/AuthorizationService.cs

```diff
@@ -50,15 +50,18 @@ public async Task<bool> IsUserAuthorizedToManageAlert(ClaimsPrincipal user, Aler
             }
 
             var userTbServiceCodes = (await _userService.GetTbServicesAsync(user)).Select(s => s.Code).ToList();
+            var userPhecCodes = (await _userService.GetPhecCodesAsync(user)).ToList();
             if (alert is TransferAlert transferAlert)
             {
-                return userTbServiceCodes.Contains(transferAlert.TbServiceCode);
+                return userTbServiceCodes.Contains(transferAlert.TbServiceCode)
+                       || userPhecCodes.Contains(transferAlert.TbService.PHECCode);
             }
 
             if (alert.NotificationId != null)
             {
-                return userTbServiceCodes.Contains(
-                    (await _notificationRepository.GetNotificationAsync((int)alert.NotificationId)).HospitalDetails.TBServiceCode);
+                var notification = await _notificationRepository.GetNotificationAsync((int)alert.NotificationId);
+                return userTbServiceCodes.Contains(notification.HospitalDetails.TBServiceCode)
+                    || userPhecCodes.Contains(notification.HospitalDetails.TBService.PHECCode);

```

Comment:
Could we add any tests for this extra logic?

Reply:
There are currently no tests for the auth service at all, so I'll write some yes. 
To answer your question about showing the transfers, regional users should already be able to see them: 
![image](https://user-images.githubusercontent.com/78739678/145555396-22443394-9242-4680-ad33-9fe4b69ec0e1.png)